templates/base/eslintrc

```diff
@@ -1,9 +1,19 @@
-{
+{    "ecmaFeatures" : {
+    "modules" : true
+     },
+
      "env": {
         "browser": true,
         "node": true,
         "es6": true
     },
+
+    /* Add scanJS rules plugin. */
+    "plugins" : [
+    "scanjs-rules",
```

Comment:
These plugins also need to be added to the package.json file so that they get installed when the project is setup up. (By "these" I mean "no-unsafe-innerhtml" and "scanjs-rules")